javascript/javascript2/week1/hay-bay/hyBayHelpers.js

```diff
@@ -0,0 +1,72 @@
+/* DONT MODIFY ANY OF THIS CODE!!!*/
+
+window.getAvailableProducts = function () {
+  function getRandomInt(min, max) {
+    min = Math.ceil(min);
+    max = Math.floor(max);
+    return Math.floor(Math.random() * (max - min + 1)) + min;
```

Comment:
this file should not be modified. I see that you haven't made any changes , but this file still shows up in your commit.